### ISSUE TITLE
[Accessibility] Example card name is not visible properly when the page is resized at 400%

### ISSUE THREAD

**Issue by dollykumari5** [OfficeDev/office-ui-fabric-react#9712]
### Environment information

OS: Version: 1903 (OS Build 18362.175) 
Browser: Edge 
Screen Reader: Narrator

### Repro steps

1. In Edge, resize the page to 400%
2. Open https://fabricweb.z5.web.core.windows.net/oufr/7.7.0/#/examples/calendar
3. Navigate to one of the Inline Calendar headings under Variants using tab key.

#### Actual behavior:

The example title is not fully visible 

![image](https://user-images.githubusercontent.com/5864305/60839631-77d85f00-a182-11e9-94d6-f5de28e9434a.png)

#### Expected behavior:

The example title should be fully visible.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: Calendar
 
[More details here](https://microsoft-my.sharepoint.com/:x:/p/v-amaver/EU_G4tOn-atHqJxnH9iVzKMB0gI7E7TZ4VGpG9N9nHuorg?e=UFZtU3)

**Comment by khmakoto** [comment]
Hi @dollykumari5, thanks for filing this issue! I can confirm this is indeed happening, though it is not specific to the `Calendar` component and seems to be more of a website spread issue. I'll take a look at it next week.

cc'ing @ecraig12345 so that you're aware of the issue.

**Comment by ecraig12345** [comment]
@dollykumari5 For future issues, please include the relevant info inline rather than linking (and don't delete the issue template). This is way more helpful both for the person investigating the issue and anyone else from the community (outside Microsoft) who might encounter a similar issue. I'm updating the issue body to include this info.

**Comment by ecraig12345** [comment]
@jordandrako Do you have a suggestion for how to fix this issue with the example cards?

**Comment by jordandrako** [comment]
Could we use a CommandBar and put those dropdowns in an overflow menu?
